src/app/shared/idle-modal/idle-modal.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <div class="modal-header">{{ "idle-modal.header" | translate }}
+    <button type="button" class="close" (click)="closePressed()" aria-label="Close">
+      <span aria-hidden="true">×</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <p>{{ "idle-modal.info" | translate:{timeToExpire: timeToExpire} }}</p>
+  </div>
+  <div class="modal-footer">
+    <button type="button" class="cancel btn btn-danger" (click)="logOutPressed()" aria-label="Log out">
+      <i class="fas fa-times"></i> {{ "idle-modal.log-out" | translate }}
+    </button>
+    <button type="button" class="confirm btn btn-outline-secondary" (click)="extendSessionPressed()" aria-label="Extend session" ngbAutofocus>
+      {{ "idle-modal.extend-session" | translate }}
```

Comment:
I'd add a refresh icon here and it'd appear as the main action in my opinion, so I'd use the btn-primary class instead of btn-outline-secondary.